scripts/create-diagrams.py

```diff
@@ -115,47 +113,35 @@ def save_figure(fig: go.Figure, filepath: str) -> None:
     )
 
 
-def add_html_header_menu(
-    filepath: str, all_packages: list[str], plotly_div_id: str = "plotly_div_id"
-) -> None:
+def add_html_header_menu(filepath: str, plotly_div_id: str = "plotly_div_id") -> None:
     """Replace plotly's opening HTML-div element with itself and an additional
-       menu so that you can navigate from any package to any other package
-       without needing to visit the index again.
+       menu so that you can navigate to different pages.
 
     Args:
         filepath (str): HTML file in which to do the replacement.
-        all_packages (str]): All the packages names for which to generate a menu entry
         plotly_div_id (str, optional): Plotly's HTML div's ID. Defaults to "plotly_div_id".
     """
     replace_me = f'<div id="{plotly_div_id}"'
 
+    last_updated = datetime.today().strftime("%c")
+
     file = Path(filepath)
-    header_menu = '<div id="headermenu">Build-Stats by package: '
-    header_menu += " | ".join(
-        [
-            '<a href="fig-{package_name}.html">{package_name}</a> '.format(
-                package_name=package_name
-            )
-            for package_name in all_packages
-        ]
-    )
-    header_menu += (
-        ' | <a href="fig-combined-standalone.html">llvm+clang+compiler-rt+libomp</a>'
-    )
-    header_menu += ' | <a href="fig-big-merge.html">llvm (big-merge)</a>'
+    header_menu = '<div id="headermenu">Build-Stats: '
+    header_menu += ' <a href="index.html">llvm (big-merge)</a>'
     header_menu += ' | <a href="fig-pgo.html">llvm (pgo)</a>'
+    header_menu += f" <small>(Last updated: {last_updated})</small>"
     header_menu += "</div>"
     header_menu += replace_me
 
     file.write_text(file.read_text().replace(replace_me, header_menu))
 
 
 # %%
-def prepare_data(filepath: str = "build-stats.csv") -> pd.DataFrame:
+def prepare_data(filepath: str = "build-stats-big-merge.csv") -> pd.DataFrame:
```

Comment:
nit: Remove default value? You don't make use of it.

Reply:
Done in 3c20f62.